### PR TITLE
PAY-1442: Pass version info in checkout and fixed for #143

### PR DESCRIPTION
**Pull request by ramth05**
PAY-1442: Pass version info in checkout and fixed for #143: Magento 2.6 Can't Accept Checkout Agreements Before Routing to razorpay.

### CODE REVIEW

File: view/frontend/web/js/view/payment/method-renderer/razorpay-method.js [hunk 1]
```diff
@@ -173,6 +178,10 @@ define(
                         name: this.user.name,
                         contact: this.user.contact,
                         email: this.user.email
+                    },
+                    _: {
+                        integration: 'Magento',
```
Comment (harman28) [inline view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: Should be lower case

File: view/frontend/web/js/view/payment/method-renderer/razorpay-method.js [hunk 2]
```diff
@@ -173,6 +178,10 @@ define(
...
+                        integration_version: data.maze_version,
```
Comment (harman28) [inline view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: This will send magento version. We want to send razorpay-magento version.

Comment (harman28) [reply view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: Actually it's probably useful to send both. Damn. Will discuss with @umanghome how we can do that.

File: view/frontend/web/js/view/payment/method-renderer/razorpay-method.js (hunk 1 above)
Comment (ramth05) [reply view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: ok

File: view/frontend/web/js/view/payment/method-renderer/razorpay-method.js (hunk 2 above)
Comment (ramth05) [reply view/frontend/web/js/view/payment/method-renderer/razorpay-method.js]: Added parent (framework) version as well.

**Pull request merged**